src/traces/cone/attributes.js

```diff
@@ -112,20 +112,26 @@ var attrs = {
         editType: 'calc',
         dflt: 'scaled',
         description: [
-            'Sets the mode by which the cones are sized.',
-            'If *scaled*, `sizeref` scales such that the reference cone size',
-            'for the maximum vector magnitude is 1.',
-            'If *absolute*, `sizeref` scales such that the reference cone size',
-            'for vector magnitude 1 is one grid unit.'
+            'Determines whether `sizeref` is set as a *scaled* (i.e unitless) scalar',
+            '(normalized by the max u/v/w norm in the vector field) or as',
+            '*absolute* value (in unit of velocity).'
```

Comment:
I wouldn't use the word "velocity" here - you can show many other quantities as vector fields (electric fields, as just one common example). Just say "in the same units as the vector field" or some such.